docs/antora.yml

```diff
@@ -1,3 +1,3 @@
 name: opendevstack
-version: 4.x
-
+version: 5.x
```

Comment:
This shouldn’t be merged into 4.x. It has to be distinct.

Reply:
It is fixed alrady in master and 4.x branch